src/test/java/HomeWork18.java

```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeWork18 extends BaseTest{
+
+    @Test
+    public void playSong () {
+
+        login("elzat.nurmanbetova@testpro.io", "kochkor123-Q!");
+
+    }
+
+    public void clickPlay() {
+        WebElement playNextBtn = driver.findElement(By.xpath("//i[@data-testid='play-next-btn']"));
+        WebElement playBtn = driver.findElement(By.xpath("//span[@data-testid='play-btn']"));
+        playNextBtn.click();
+
+        playBtn.click();
+
+    }
+
+    public boolean isSongPlays (){
+        WebElement soundBar = driver.findElement(By.xpath("img[alt='Sound bars']"));
+        return soundBar.isDisplayed();
+    }
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**

For example 

![18](https://github.com/Test-Pro-Bootcamp/A57-QA-Automation-Framework/assets/104761482/20ba7772-6956-43b8-9878-5d9a0d25b7cb)